Vostok.Logging.File/FileLog.cs

```diff
@@ -95,14 +102,22 @@ internal FileLog(IMultiFileMuxer muxer, Func<FileLogSettings> settingsProvider)
             eventsLost = new AtomicLong(0);
 
             filePathProvider = new CachingTransform<FileLogSettings, FilePath>(
-                settings => new FilePath(settings.FilePath), preventParallelProcessing: false);
+                settings => new FilePath(settings.FilePath),
+                preventParallelProcessing: false);
+
+            Instances.Add(this);
```

Comment:
Should the log also remove itself from `Instances` after a successful `Dispose` call?